tests/test-ci-build-props.sh

```
@@ -0,0 +1,32 @@
+    exit 1
+fi
+
+echo "Testing ContinuousIntegrationBuild property (GITHUB_ACTIONS=true)..."
+GH_BUILD=$(GITHUB_ACTIONS=true dotnet msbuild ../src/Zipper.csproj -getProperty:ContinuousIntegrationBuild)
+if [[ "$GH_BUILD" != "true" ]]; then
+    echo "FAIL: ContinuousIntegrationBuild is not true when GITHUB_ACTIONS=true. Got '$GH_BUILD'"
+    exit 1
+fi
+
+echo "Testing ContinuousIntegrationBuild property (no CI)..."
+NO_CI_BUILD=$(CI= GITHUB_ACTIONS= dotnet msbuild ../src/Zipper.csproj -getProperty:ContinuousIntegrationBuild)
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)